docs/antrea-ipam.md

```diff
@@ -218,6 +218,34 @@ metadata:
 A StatefulSet Pod's IP will be kept after Pod restarts, when the IP is allocated from the
 annotated IPPool.
 
+### IPAM behaviors
+
+When `AntreaIPAM` is enabled, `antrea-controller` will watch IPPool CRs and StatefulSets
+from K8s apiserver, and `antrea-agent` will register IPAM driver to CNI server.
```

Comment:
Probably, `kube-apiserver`.

IPAM driver -> the IPAM driver

Reply:
Consider, "activate the Antrea IPAM driver". "register to CNI server" is too much internal details.

Reply:
Fixed.